libs/core/src/lib/fixed-card-layout/fixed-card-layout.component.ts

```diff
@@ -63,6 +75,92 @@ class CardDropped {
 
 type CardColumn = CardDefinitionDirective[];
 
+@Directive({ selector: '[fdGroup]' })
+export class DndGroup {
+    /** @hidden */
+    _onDndItemFocus$ = new Subject<[number, number]>();
+
+    constructor(public cdr: ChangeDetectorRef) {}
+
+    /** add focus after moving elements with keyboard */
+    focusDndItem(groupIndex: number, itemIndex: number): void {
+        this.cdr.detectChanges();
+        this._onDndItemFocus$.next([groupIndex, itemIndex]);
+    }
+}
+
+@Directive({ selector: '[fdItem]' })
+export class DndItem implements OnInit {
+    /** item index in group(column) */
+    @Input()
+    itemIndex: number;
+  
+    /** group(column) index */
+    @Input()
+    groupIndex: number;
+
+    constructor(
+        private dndGroup: DndGroup, 
+        private elementRef: ElementRef, 
+        private cardLayout: FixedCardLayoutComponent
+    ) {}
+
+    /** @hidden */
+    ngOnInit(): void {
+        this.dndGroup._onDndItemFocus$.subscribe(([groupIndex, itemIndex]) => {
+            if (this.groupIndex === groupIndex && this.itemIndex === itemIndex) {
+              this.elementRef.nativeElement.focus();
+            }
+        });
+    }
+    
+    /** disabled possibility to move card */
+    @HostListener('keyup', ['$event'])
+    onKeyUp(event: KeyboardEvent): void {
```

Comment:
`@hidden`? If yes please also prefix method name with `_`. Same for other methods.

Reply:
done